nilearn/decoding/fista.py

```diff
@@ -231,7 +236,7 @@ def mfista(f1_grad, f2_prox, total_energy, lipschitz_constant, w_size,
             #
             # For this reason, we rely more on the linesearch-like
             # strategy to set the dgap_tol
-            dgap_tol = abs(energy_delta) / (i + 1.)
+            dgap_tol = abs(energy_delta)
```

Comment:
makes sense.

While we're at it, should we think about relaxing the dual gap tolerance again by line searching over it with warm starts? Probably for another PR.